vhdlparser/JavaCC.h

```diff
@@ -35,12 +39,13 @@ class ReaderStream {
 const JAVACC_CHAR_TYPE EMPTY[] = { 0 };
 
 #ifndef MAX
-#define MAX(a,b) ((a)>=(b)?(a):(b))
+#define MAX(a,b) (a>=b?a:b)
 #endif
 #ifndef MIN
-#define MIN(a,b) ((a)<=(b)?(a):(b))
+#define MIN(a,b) (a<=b?a:b)
```

Comment:
I think it is better to have the round brackets as e.g:
```
MAX (3+5,7)
```
will otherwise give different results.